<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemstatusPanel</class>
 <widget class="QDockWidget" name="SystemstatusPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>377</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>326</width>
    <height>377</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>800</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="windowTitle">
   <string>System Status</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>System Status</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item row="2" column="0">
        <widget class="QLabel" name="LBL_ParticleNumber">
         <property name="text">
          <string>Displayed Particles</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="LBL_ParamFile">
         <property name="text">
          <string>Filter Parameter File</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="particleNumber">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="LBL_GPUmemory">
         <property name="text">
          <string>GPU Memory</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="parameterFilename">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Filterinfo | window title</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QProgressBar" name="cpu_useage_bar">
         <property name="toolTip">
          <string>Currently used CPU Memory</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QProgressBar::chunk {
     background-color: #25cc25;
     width: 1px;
 }

 QProgressBar {
     border: 1px solid lightgrey;
     border-radius: 2px;
     text-align: center;
 }</string>
         </property>
         <property name="value">
          <number>46</number>
         </property>
         <property name="format">
          <string>15718/12286MB</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="LBL_CPUmemory">
         <property name="text">
          <string>CPU Memory</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QProgressBar" name="gpu_usage_bar">
         <property name="toolTip">
          <string>Currently used GPU Memory</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QProgressBar::chunk {
     background-color: #3add36;
     width: 1px;
 }

 QProgressBar {
     border: 1px solid lightgrey;
     border-radius: 2px;
     text-align: center;
 }</string>
         </property>
         <property name="value">
          <number>50</number>
         </property>
         <property name="format">
          <string>4096/8192MB</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>Sampling </string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item row="0" column="1">
        <widget class="QSpinBox" name="samplingInterval">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::PlusMinus</enum>
         </property>
         <property name="suffix">
          <string>ms</string>
         </property>
         <property name="maximum">
          <number>999999999</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>5000</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="disableSampling_chk">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Hold till complete</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="LBL_SamplingInterval_2">
         <property name="text">
          <string>Step Interval</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="LBL_disableSampling">
         <property name="text">
          <string>Transfer Function Edit</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="setParam_btn">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
